UI/cypress/e2e/postcode.feature

```diff
@@ -22,7 +22,7 @@ Feature: User enters postcode to begin search
 
   Scenario: user clicks quality assured tuition partner details summary
     Given a user has started the 'Find a tuition partner' journey
-    When they click 'What is a quality assured tuition partner?'
+    When they click 'What is a quality-assured tuition partner?'
     Then the quality assured tuition partner details are shown
 
```

Comment:
The presence of the two new options boxes should be verified with end to end tests as should their linking to the AM and SLT pages